src/ua_parser/core.py

```diff
@@ -0,0 +1,328 @@
+import abc
+import re
+from dataclasses import dataclass, fields
+from enum import Flag, auto
+from typing import Literal, Optional, Tuple, List
+
+__all__ = [
+    "UserAgent",
+    "OS",
+    "Device",
+    "ParseResult",
+    "DefaultedParseResult",
+    "Parser",
+    "Domain",
+    "PartialParseResult",
+    "Matchers",
+    "UserAgentMatcher",
+    "OSMatcher",
+    "DeviceMatcher",
+]
```

Comment:
Might be good to order this since black keeps reformatting it into a flat list?

Alternatively re-group it and mark it as `fmt: off`.